### DISCUSSION TITLE
2D vs. 3D display of buildings

### DISCUSSION THREAD

**Discussion by moving-bits** [mapsforge/vtm#1124] [Q&A] [answered]
Same map ("Berlin OAM offline .map file") with same theme ("Elevate.zip") displays buildings in flat 2D with Mapsforge, but with pseudo-3D with VTM.

Is there a setting to optionally switch VTM to 2D as well?

**Comment by devemux86** [comment, marked as answer]
3D buildings are controlled by the BuildingLayer, so can add it if they are needed or not.

**Reply by moving-bits** [reply]
Ah, thanks, that was what I was looking for.
